tools/test.py

```diff
@@ -111,7 +111,11 @@ def main():
     if args.cfg_options is not None:
         cfg.merge_from_dict(args.cfg_options)
 
-    set_random_seed(1, deterministic=True)
+    if cfg.get('cudnn_benchmark', False):
+        torch.backends.cudnn.benchmark = True
+    if cfg.get('seed', None) is not None:
```

Comment:
add 
`# set random seeds. Force setting fixed seed and deterministic=True in SOT`
    `# configs`